src/dfktools/quests/quest_v1.py

```diff
@@ -22,6 +23,9 @@ def complete_quest(self, hero_id, private_key, nonce, gas_price_gwei, tx_timeout
     def parse_complete_quest_receipt(self, tx_receipt):
         return quest_core_v1.parse_complete_quest_receipt(tx_receipt, self.rpc_address)
 
+    def parse_quest_rewards(self, quest_results):
+        return quest_core_v1.parse_quest_rewards(self.realm, quest_results)
```

Comment:
see this comment https://github.com/0rtis/dfktools/pull/35/commits/8164e1d997d2b56d8da6914530d9d2899f47e9a7#r922920101